pkg/datamover/datamover.go

```diff
@@ -137,6 +138,12 @@ func DeleteTempVSClass(backupName string, tempVS snapshotv1listers.VolumeSnapsho
 	tempVSClassName := fmt.Sprintf("%s-snapclass", backupName)
 	tempVSClass, err := tempVS.Get(tempVSClassName)
 	if err != nil {
+		// ignore is not found err as it is possible to create data mover
+		// backup without an existing PVC
+		// in which case this VSClass will not exist
+		if apierrors.IsNotFound(err) {
+			return nil
```

Comment:
seems fine to me IF the only errors we need to be concerned about are apierrors.

Reply:
AFAIK for this particular issue this should be the only error we're concerned about